ocaml/utils/load_path.ml

```diff
@@ -175,6 +203,10 @@ let init ~auto_include ~visible ~hidden =
   reset ();
   visible_dirs := List.rev_map (Dir.create ~hidden:false) visible;
   hidden_dirs := List.rev_map (Dir.create ~hidden:true) hidden;
+  List.iter (fun (libloc : Clflags.Libloc.t) ->
```

Comment:
I'm not sure if that's the place where we want to get libloc from command line arguments. It looks like a better approach would be to pass it as argument to `init`, but it's not clear to me in which of `Load_path.init` calls we should include `libloc`

Reply:
I'm fine with doing it here